packages/flutter_goldens/lib/flutter_goldens.dart

```diff
@@ -131,15 +131,15 @@ abstract class FlutterGoldenFileComparator extends GoldenFileComparator {
   /// Calculate the appropriate basedir for the current test context.
```

Comment:
Driveby comment: you may want to add to the documentation here explaining what suffix is and why it is needed.